server/system/service/attachment.go

```diff
@@ -198,8 +200,49 @@ func (svc attachment) CreateApplicationAttachment(ctx context.Context, name stri
 	return att, svc.recordAction(ctx, aaProps, AttachmentActionCreate, err)
 }
 
+func (svc attachment) CreateAuthAttachment(ctx context.Context, name string, size int64, fh io.ReadSeeker, labels map[string]string) (att *types.Attachment, err error) {
+	var (
+		aaProps       = &attachmentActionProps{}
+		currentUserID = intAuth.GetIdentityFromContext(ctx).Identity()
+	)
+
+	// check the file type
+	ext := filepath.Ext(name)
+	if ext != ".jpg" && ext != ".jpeg" && ext != ".png" {
+		return nil, AttachmentErrInvalidAvatarFileType()
+	}
+
+	//check the file size
+	if size > 3000000 {
```

Comment:
Try to avoid magic numbers.
Should we keep file size limitations consistent and have it configurable?